### ISSUE TITLE
[Question] Is it safer to use Startpage instead of Google

### ISSUE THREAD

**Issue by RaptaG** [privau/searxng#21]
Is SearXNG proxying the Google results for safety? How well is it done if yes?
Is it safer to use Startpage (Startpage is proxied Google results)?

**Comment by vojkovic** [comment]
Hi Thank-you for your issue,

SearXNG proxies all queries regardless of the engine. So if you wanted to query startpage it would be like this:

You <--> Priv.au <--> Startpage <--> Google Search

Google can see Startpage's IP address and your query is mixed with thousands of other startpage queries per hour. Google doesn't know much about you except that your query came from Startpage. Although, startpage knows that your query came from Priv.au.

If you query Google directly through priv.au, it looks more like this:

You <--> Priv.au <--> Google Search

In this case, Google can see that someone using Priv.au is searching. Your result might only be mixed with a few hundred per hour or however many people use my instance, I don't really know. In theory, Google could know more about you because less people use Priv.au rather than startpage, but I don't believe there to be any serious tangible privacy benefits for using one rather than the other. 

TLDR: I don't think it makes much of a difference.

**Comment by RaptaG** [comment]
Google has faster response time so I'll use that, thanks!
